### PR TITLE
vm_creator: output VM ips in a bash-compatible format

### PR DESCRIPTION
**Pull request by elad661**
vm_creator: output VM ips in a bash-compatible format

\+ Some code style fixes, and changed the default value of the node count from 3 to 2.

### CODE REVIEW

File: cm_ovirt_vm_creator.py [hunk 1]
```diff
@@ -77,7 +75,7 @@ def do_work(args):
         for idx in range(1, args.nodes+1):
             cluster_nodes.append(vm_name_template.format(name_prefix=args.name_prefix,
                                                          node_type="compute", i=idx))
-        print (cluster_nodes)
+        print(cluster_nodes, file=sys.stderr)
```
Comment (moolitayer) [inline cm_ovirt_vm_creator.py:78]: why stderr?

File: cm_ovirt_vm_creator.py [hunk 2]
```diff
@@ -87,42 +85,49 @@ def do_work(args):
...
-    nodes = []
     for node in vm_iterator(cluster_nodes):
         node_name = node.get().name
-        print (node_name)
-        for dev in node.reported_devices_service().list():
-            if dev.name == 'eth0':
-                for ip in dev.ips:
-                    if ip.version == types.IpVersion.V4:
-                        vm_dict[node_name] = ip.address
+        print(node_name, file=sys.stderr)
```
Comment (moolitayer) [inline cm_ovirt_vm_creator.py:93]: here too

File: cm_ovirt_vm_creator.py (hunk 1 above)
Comment (elad661) [reply cm_ovirt_vm_creator.py:78]: so stdout outputs something that you can save to a file and later `source` in bash, and all other human-friendly, not bash-compatible output will be in stderr.

Comment (moolitayer) [reply cm_ovirt_vm_creator.py:78]: slight abuse of stderr. You could pass a file name to cm_ovirt_vm_creator and print the out to that file.

**Pull request merged**